app.js

```diff
@@ -0,0 +1,107 @@
+"use strict";
+
+var restify = require('restify');
+var Sequelize = require('sequelize');
+
+var sequelize = new Sequelize('mvctodo', 'postgres', 'nicopostgres', {
+    host: "localhost",
+    port: 5432,
+    dialect: 'postgres'
+});
+
+var todoentries = sequelize.define('todoentries', {
+    title: {
+        type: Sequelize.STRING,
+        field: 'title'
+    },
+    completed: { type: Sequelize.BOOLEAN }
+},
+    {
+        createdAt: false,
+        updatedAt: false,
+        freezeTableName: true
+    }
+    );
+
+function CreateTodo(req, res, next) {
+    todoentries.sync({ force: true }).then(function () {
+        todoentries.build({
+            title: req.body.title,
+            completed: false
+        })
+            .save()
+            .then(function (test) {
+                res.status(201);
+                res.charSet("utf-8");
+                res.json(test.dataValues);
+            });
+    });
+    next();
+}
+
+function ListTodos(req, res, next) {
+    todoentries.findAll()
+        .then(function (rows) {
+            res.charSet("utf-8");
+            res.json(rows);
+        });
+    next();
+}
+
+function ArchiveTodos(req, res, next) {
+    todoentries.destroy({ where: { completed: true }})
+        .then(function () {
+            res.send(204);
+        });
+    next();
+}
+
+function GetTodo(req, res, next) {
+    todoentries.findAll({
+        where: {
+            id: req.params.id,
+        }
+    }).then(function (rows) {
+        res.charSet("utf-8");
+        res.json(rows[0]);
+    });
+    next();
+}
+
+function UpdateTodo(req, res, next) {       //STILL NEEDS TO UPDATE
+    todoentries.update({completed: true}, {where: {id: req.params.id}})
```

Comment:
Rather than run an update query on the DB, you're better off fetching the model and making a change on the model. Focus on objects, not queries :)